extensions/Khronos/KHR_mesh_compression/schema/attribute.KHR_draco_geometry_compression.schema.json

```diff
@@ -0,0 +1,27 @@
+{
+    "$schema" : "http://json-schema.org/draft-04/schema",
+    "title" : "attribute",
+    "type" : "object",
+    "properties" : {
+        "semantic" : {
+          "type" : "string",
+          "description" : "Mesh attribute semantic. e.g. POSITION, NORMAL and TEXCOORD.",
```

Comment:
Probably should add that all restrictions on semantics names from the core spec still apply (i.e., `TEXCOORD_%d` instead of just `TEXCOORD`).